fiftyone/core/session.py

```diff
@@ -356,6 +350,40 @@ def __init__(
             self.open()
             return
 
+    def import_desktop():
+
+        """Imports the `fiftyone-desktop` package and verifies that the correct
+        version is installed.
+        """
```

Comment:
You can make this a private method in the module (outside of `Session`) at the end of `session.py` after
`def _display_colab():`. No docstring is necessary.

Reply:
```py
def _import_desktop():
    try:
        import fiftyone.desktop
    except ImportError as e:
        raise ValueError(
            "You must `pip install fiftyone[desktop]` in order to launch the "
            "desktop App"
        ) from e

    # Get `fiftyone-desktop` requirement for current `fiftyone` install
    fiftyone_dist = pkg_resources.get_distribution("fiftyone")
    requirements = fiftyone_dist.requires(extras=["desktop"])
    desktop_req = [r for r in requirements if r.name == "fiftyone-desktop"][0]

    desktop_dist = pkg_resources.get_distribution("fiftyone-desktop")

    if not desktop_req.specifier.contains(desktop_dist.version):
        raise ValueError(
            "fiftyone==%s requires fiftyone-desktop%s, but you have "
            "fiftyone-desktop==%s installed.\n"
            "Run `pip install fiftyone[desktop]` to install the proper "
            "desktop package version" % (
                fiftyone_dist.version,
                desktop_req.specifier,
                desktop_dist.version
            )
        )
```